modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Sequence.scala

```diff
@@ -12,6 +13,10 @@ case class Sequence[+A](id: String, metadata: SequenceMetadata, steps: List[Step
 
 object Sequence {
 
+  type Id = String
```

Comment:
Most likely, but I wanted to introduce a type alias to keep the engine isolated from a possible change in the type used to identify sequences (which I think is something planned for the ODB).